extension/war/src/main/webapp/groovy/social/webui/profile/UIUserActivitiesDisplay.gtmpl

```diff
@@ -91,13 +90,6 @@
 %>
 <div id="$uicomponent.id" class="uiUserActivitiesDisplay uiActivitiesDisplay clearfix"> <!-- use js to add beside one class that named "notSeen" if having some new updates  -->
 <% if (uicomponent.isWelcomeActivity()) { %>
-  <div class="composerHintCont">
-	<span class="arrowLeft"></span>
-	<a id="ComposerHintCloseBtn" href="javascript:void(0);" class="iconBorder"><i class="uiIconClose uiIconLightGray"></i></a>								
-	<div class="hint">
-		 $composer_Hint_Text	
-	</div>
-  </div>
```

Comment:
We should also delete the related js script : https://github.com/Meeds-io/social/pull/97/files#diff-1b43fc6c90c76d37eae83b7a4a76b8e5L86-L90